src/main/java/emissary/core/channels/SeekableByteChannelHelper.java

```diff
@@ -158,7 +158,7 @@ public static int getFromInputStream(final InputStream inputStream, final ByteBu
         // Read direct into buffer's array if possible, otherwise copy through an internal buffer
         final int bytesToRead = byteBuffer.remaining();
         if (byteBuffer.hasArray()) {
-            final int bytesRead = inputStream.read(byteBuffer.array(), byteBuffer.position(), bytesToRead);
+            final int bytesRead = inputStream.read(byteBuffer.array(), byteBuffer.arrayOffset() + byteBuffer.position(), bytesToRead);
```

Comment:
What was flagged to initiate this change?

Reply:
I think it's a good fix to make, but I didn't see it in the list on the originating ticket. In the future, feel free to add notes about stuff like this to the PR description to give reviewers a heads up. That can potentially reduce confusion on if the change was intentional.